src/System.Diagnostics.Contracts/src/System.Diagnostics.Contracts.builds

```diff
@@ -4,9 +4,6 @@
   <ItemGroup>
     <Project Include="System.Diagnostics.Contracts.csproj" />
     <Project Include="System.Diagnostics.Contracts.csproj">
-      <TargetGroup>net46</TargetGroup>
```

Comment:
I don't fully understand these comments, so I can't say. I rebased by changes against this and had kept all of the comments around facades being in-box. My change in this area just removed the "facade" part of the path from the Project Include, but everything else I left the same.